Day1-adding_to_k/Brandon/adding.py

```diff
@@ -0,0 +1,28 @@
+# Given a list of numbers and a number k, return whether any two numbers from the list add up to k.
+# For example, given [10, 15, 3, 7] and k of 17, return true since 10 + 7 is 17.
+# Bonus: Can you do this in one pass?
+
+from copy import deepcopy
```

Comment:
@adamhursey you can comment on specific lines here. So deep copy makes a full blown copy of the list. So the way that I am working with the list it would delete one object each time I am iterating.